### PR TITLE
Serve nextjs via node

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard4**
## Summary

Serve nextjs via node

### CODE REVIEW

File: packages/app/src/domain/topical/sitemap/utils.tsx [hunk 1]
```diff
@@ -155,6 +155,10 @@ export function useDataSitemap(
           text: siteText.besmettelijke_personen.titel_sidebar,
           href: reverseRouter.nl.besmettelijkeMensen(),
         },
+        {
+          text: siteText.besmettelijke_personen.titel_sidebar,
+          href: reverseRouter.nl.besmettelijkeMensen(),
+        },
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/domain/topical/sitemap/utils.tsx:161]: merge conflict mistake? appears to be duplicate now

File: packages/app/src/domain/variants/variants-table-tile/index.tsx [hunk 2]
```diff
@@ -53,6 +53,7 @@ export function VariantsTableTile({
   } | null;
   children?: ReactNode;
 }) {
+  const { siteText, formatDateSpan } = useIntl();
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/domain/variants/variants-table-tile/index.tsx:56]: Can you validate these variables are used in this component? With the diff it looks like they're unused, might be merge conflict mistake?

File: packages/app/src/pages/internationaal/index.tsx [hunk 3]
```diff
@@ -1,30 +0,0 @@
-import { InternationalLayout } from '~/domain/layout/international-layout';
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/pages/internationaal/index.tsx:1]: this file is still present in the develop branch, delete on purpose or is it another merge conflict gone wrong..?

File: packages/app/src/pages/sitemap.xml.tsx [hunk 4]
```diff
@@ -1,14 +1,54 @@
...
+            <loc>{[origin, page.path].join('')}</loc>
+            <priority>{page.priority}</priority>
+          </url>
+        );
+      })}
+    </urlset>
+  );
+};
+
+export const getServerSideProps = async ({
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/pages/sitemap.xml.tsx:32]:
```suggestion
export async function getServerSideProps ({
```

File: packages/app/src/pages/sitemap.xml.tsx [hunk 5]
```diff
@@ -1,14 +1,54 @@
...
-import { logError } from '../utils';
-require('dotenv').config({ path: '../app/.env.local' });
+import { renderToStaticMarkup } from 'react-dom/server';
+import { features } from '~/config';
+import { gmData } from '~/data/gm';
+import { vrData } from '~/data/vr';
+
+const SitemapIndex = () => null;
+
+const Sitemap = ({
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/pages/sitemap.xml.tsx:11]:
```suggestion
export default function Sitemap({ ... }) {
```

File: packages/app/src/static-props/create-get-static-props.tsx [hunk 6]
```diff
@@ -138,7 +138,10 @@ export function createGetStaticProps(
 
     const props = promisedProps.reduce((res, val) => ({ ...res, ...val }), {});
 
-    return { props };
+    return {
+      props,
+      revalidate: 1,
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/static-props/create-get-static-props.tsx:143]: isn't this a bit on the low side?

File: packages/app/vercel-check-nodejs-deploy.sh [hunk 7]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+echo "VERCEL_GIT_COMMIT_REF: $VERCEL_GIT_COMMIT_REF"
+
+if [[ "$VERCEL_GIT_COMMIT_REF" == *"topic/production-on-node"* ]]; then
```
Comment (VWSCoronaDashboard10) [inline packages/app/vercel-check-nodejs-deploy.sh:4]: current branch is named something else, does that matter?

File: packages/cli/package.json [hunk 8]
```diff
@@ -30,7 +30,7 @@
     "validate-last-values": "yarn workspace @corona-dashboard/common build && ts-node src/scripts/validate-last-values.ts",
     "validate-features": "yarn workspace @corona-dashboard/common build && ts-node src/scripts/validate-features.ts",
     "generate-typescript": "yarn workspace @corona-dashboard/common build && ts-node src/scripts/generate-data-types.ts",
-    "generate-sitemap": "ts-node src/scripts/generate-sitemap.ts",
+    "generate-sitemap": "echo 'generate-sitemap is deprecated' && exit 1",
```
Comment (VWSCoronaDashboard10) [inline packages/cli/package.json:33]: We can remove this line. Calling a script which does not exist will also throw an error.

File: packages/cms/package.json [hunk 9]
```diff
@@ -11,7 +11,7 @@
     "test:ci": "yarn run test",
     "compile": "tsc --pretty --noEmit",
     "lint": "echo '@TODO add eslint'",
-    "sync-assets": "ts-node -r dotenv-flow/config scripts/sync-assets.ts",
+    "sync-assets": "echo 'Sync assets is deprecated' && exit 1",
```
Comment (VWSCoronaDashboard10) [inline packages/cms/package.json:14]: same here

File: packages/cms/src/schemas/documents/lokalize-text.tsx [hunk 10]
```diff
@@ -232,3 +232,100 @@ Een "accent grave", ook wel bekend als "backtick", is ook mogelijk.
     </div>
   );
 }
+
+function LokalizeTextDescription() {
```
Comment (VWSCoronaDashboard10) [inline packages/cms/src/schemas/documents/lokalize-text.tsx:236]: I believe in the develop branch this component has been moved to a separate file. Merge conflit..?

**Review by VWSCoronaDashboard10 — COMMENTED** [review]
I'm afraid quite a lot of merge conflicts have gone wrong .. 😞

File: packages/cms/src/schemas/documents/lokalize-text.tsx (hunk 10 above)
Comment (VWSCoronaDashboard10) [reply packages/cms/src/schemas/documents/lokalize-text.tsx:236]: ☝️  This is not true, i've seen it moved to another file in another branch. Though it's weird that this change looks like "added code", but it is already part of develop?

File: packages/app/next.domains.config.js [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+  {
+    domain: 'coronadashboard.nl',
+    defaultLocale: 'nl',
+  },
+  {
+    domain: 'coronadashboard.rijksoverheid.nl',
+    defaultLocale: 'nl',
+  },
+  {
+    domain: 'coronadashboard.government.com',
```
Comment (VWSCoronaDashboard10) [inline packages/app/next.domains.config.js:15]: this should be 
```suggestion
    domain: 'coronadashboard.government.nl',
```

**Review by VWSCoronaDashboard10 — CHANGES_REQUESTED** [review]

File: packages/cli/package.json (hunk 8 above)
Comment (VWSCoronaDashboard8) [reply packages/cli/package.json:33]: 👍  Or, if its deprecated you would typically at first not make it throw an error so keeping it in with exit 0 / no exit declaration would also make sense to me.

File: packages/cms/package.json (hunk 9 above)
Comment (VWSCoronaDashboard8) [reply packages/cms/package.json:14]: Same for me also :)

**Review by VWSCoronaDashboard8 — APPROVED** [review]

**Pull request merged**